### ISSUE TITLE
✨ Feature Suggestion | Timelines for team member / emeriti

### ISSUE THREAD

**Issue by Mikaela** [privacytools/privacytools.io#2089]
## Description

I was wondering if there would be any point in having the time periods when people became team members or were active as team members as emeriti in the about page or would it become ranking/competitive or resemble gravestones.

* https://www.privacytools.io/about/

**Comment by LongJohn-Silver** [comment]
I say yes.

**Comment by dngray** [comment]
Yeah, this would be a good idea, we should say the period they participated!
